examples/remix/app/components/core/interface/read-functions.tsx

```
@@ -0,0 +1,178 @@
+              </div>
+            </div>
+            
+            {/* Function inputs */}
+            {hasInputs && (
+              <div className="mb-4 space-y-3">
+                {func.inputs.map((input, index) => (
+                  <div key={`${functionId}-input-${index}`} className="flex flex-col gap-1">
+                    <label className="text-sm text-muted-foreground">
+                      {input.name || `param${index}`}{' '}
+                      <span className="text-xs font-mono">({input.type})</span>
+                    </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)